### ISSUE TITLE
Missing (?) lib/App/termpub/Renderer.pm

### ISSUE THREAD

**Issue by kseistrup** [mdom/termpub#3]
Building termpub 1.05 fails with the following error message:

```
Can't open lib/App/termpub/Renderer.pm: No such file or directory
```

**Comment by apetresc** [comment]
Not sure why this is closed - it's true, the `build-standalone` script is outdated and references files that no longer exist. This is breaking the AUR `PKBUILD` too :(

**Comment by apetresc** [comment]
Oh, I see, it was fixed! My mistake - could you cut a 1.06 release with this commit so I can fix the `PKGBUILD` for it too? :)

**Comment by mdom** [comment]
I just released the fix along with some other changes.

Is there a reason why AUR uses the build-standalone script? I imagine there a canonical way to install perl libraries? Sorry, but know nothing about arch... :)
